kuma/payments/jinja2/payments/includes/payments-form.html

```diff
@@ -35,9 +35,10 @@
         <p class="legal-copy">
             Payments are not tax deductible. Your payment details will be processed by
             <a href="https://stripe.com/" rel="noopener" target="_blank">Stripe</a>.
-            You can read our
-            <a href="{{ url('payment_terms') }}" rel="noopener" target="_blank">Payment terms</a>,
-            for more information.
+            You can see more information in our
```

Comment:
"You can see more information in our [Payment terms](https://developer.mozilla.org/en-US/payments/terms), and [Privacy notice](https://www.mozilla.org/en-US/privacy/websites/)" feels awkward to me. 

I'd like "See our [payment terms](https://developer.mozilla.org/en-US/payments/terms) and [privacy policy](https://www.mozilla.org/en-US/privacy/websites/)". I've suggested it in the copy document.

Reply:
Our lawyer approved the shorter phrasing, but use "privacy notice", and confirmed the destination for the link.